app/src/main/java/org/fossasia/phimpme/share/SharingActivity.java

```diff
@@ -448,24 +451,28 @@ protected void onPostExecute(Void result) {
     }
 
     private void flickrShare() {
-        Intent intent = new Intent(getApplicationContext(),
-                FlickrActivity.class);
-        InputStream is = null;
-        File file = new File(saveFilePath);
-        try {
-            is = getContentResolver().openInputStream(Uri.fromFile(file));
-        } catch (Exception e) {
-            e.printStackTrace();
-        }
-        if (is != null) {
-            FlickrHelper f = FlickrHelper.getInstance();
-            f.setInputStream(is);
-            f.setFilename(file.getName());
+        if (Utils.checkAlreadyExist(FLICKR)) {
+            Intent intent = new Intent(getApplicationContext(),
```

Comment:
@Shubhankar please remove for  Flickr code because I am cleaning code for Flickr. Please send PR by today. In that I am doind this in different way.